src/main/resources/assets/natura/lang/de_DE.lang

```diff
@@ -0,0 +1,582 @@
+tile.natura.nether_leaves2.darkwood_fruit.name=Dunkelholzblätter
+tile.natura.nether_leaves2.darkwood_fruit.tooltip=Netherbaum\nEnthält einen Apfel
+
+tile.natura.nether_sapling.ghostwood.name=Geisterholzsetzling
+tile.natura.nether_sapling.ghostwood.tooltip=Bleich wie ein Gespenst
+tile.natura.nether_sapling.fusewood.name=Zündholzsetzling
+tile.natura.nether_sapling.fusewood.tooltip=Explosive Persönlichkeit
+tile.natura.nether_sapling.darkwood.name=Schwarzholzsetzling
+tile.natura.nether_sapling.darkwood.tooltip=Produziert kreidige Äpfel
+
+tile.natura.nether_sapling2.bloodwood.name=Blutholzsetzling
+tile.natura.nether_sapling2.bloodwood.tooltip=Feuerbeständige Bretter
+
+tile.natura.nether_planks.ghostwood.name=Geisterholzbretter
+tile.natura.nether_planks.ghostwood.tooltip=Bleich wie ein Gespenst
+tile.natura.nether_planks.bloodwood.name=Blutholzbretter
+tile.natura.nether_planks.bloodwood.tooltip=Feuerbeständige Bretter
+tile.natura.nether_planks.fusewood.name=Zündholzbretter
+tile.natura.nether_planks.fusewood.tooltip=Netherbaum\nExplodierende Persönlichkeit
+tile.natura.nether_planks.darkwood.name=Dunkelholzbretter
+tile.natura.nether_planks.darkwood.tooltip=Netherbaum
+
+tile.natura.nether_slab.ghostwood.name=Geisterholzstufe
+tile.natura.nether_slab.ghostwood.tooltip=Bleich wie ein Gespenst
+tile.natura.nether_slab.bloodwood.name=Blutholzstufe
+tile.natura.nether_slab.bloodwood.tooltip=Feuerbeständige Bretter
+tile.natura.nether_slab.fusewood.name=Zündholzstufe
+tile.natura.nether_slab.fusewood.tooltip=Netherbaum\nExplodierende Persönlichkeit
+tile.natura.nether_slab.darkwood.name=Dunkelholzstufe
+tile.natura.nether_slab.darkwood.tooltip=Netherbaum
+
+tile.natura.nether_stairs_ghostwood.name=Geisterholztreppe
+tile.natura.nether_stairs_bloodwood.name=Blutholztreppe
+tile.natura.nether_stairs_darkwood.name=Dunkelholztreppe
+tile.natura.nether_stairs_fusewood.name=Zündholztreppe
+
+tile.natura.nether_glass.soul.name=Seelenglas
+tile.natura.nether_glass.soul.tooltip=Gespenstisch
+tile.natura.nether_glass.heat.name=Hitzeglas
+
+tile.natura.nether_tainted_soil.tainted_soil.name=Verdorbener Boden
+tile.natura.nether_tainted_soil.tainted_farmland_dry.name=Verdorbener Boden
+tile.natura.nether_tainted_soil.tainted_farmland_heated.name=Verdorbener Boden
+
+tile.natura.nether_heat_sand.name=Hitzesand
+
+tile.natura.nether_berrybush_blightberry.name=Brombeerstrauch
+tile.natura.nether_berrybush_blightberry.tooltip=Ein bisschen böse\nKillerheilung
+tile.natura.nether_berrybush_duskberry.name=Waldbeerenbusch
+tile.natura.nether_berrybush_duskberry.tooltip=Ein bisschen böse\nUnsichtbare Nacht
+tile.natura.nether_berrybush_skyberry.name=Himmelsbeerenbusch
+tile.natura.nether_berrybush_skyberry.tooltip=Etwas böse\nLangsames Springen
+tile.natura.nether_berrybush_stingberry.name=Stechbeerenbusch
+tile.natura.nether_berrybush_stingberry.tooltip=Ein bisschen böse\nSchlag wie ein Lastwagen
```

Comment:
This should not be translated literally. I suggest something along "Triff da, wo es weh tut"

Reply:
Yeah, I know, but I found that translation kinda funny so I thought it might be fitting.
But I'll change it.